plugins/carbonetes-scan.yaml

```diff
@@ -5,15 +5,17 @@ metadata:
 spec:
   homepage: https://github.com/carbonetes/kubectl-carbonetes-scan
   shortDescription: Integrates container analysis directly into your cluster.
-  version: v1.0.2
+  version: v1.1.0
   description: |
     Provides complete container analysis and
     policy evaluation as a fully managed service.
     Carbonetes analyzes your container images for native
     code vulnerabilities, software composition analysis (SCA),
     license types, malware, secrets, and bill of materials.
   caveats: |
-    Docker needs to be installed on your machine to use this plugin.
+    You need to have a valid credentials at Carbonetes to execute this plugin.
+    Doesn't have any credentials yet?
+    Register here: https://carbonetes.com
```

Comment:
🤔 This seems to be a new requirement. Is that really necessary?
I think no existing plugin requires an account for a third-party site. I was looking if we explicitly require that plugins are open source, but could not find it. Let's hear more opinions.
/cc @ahmetb @chriskim06

Reply:
Ok, I think an account as prerequisite is fine. However, please make it  clear that the evaluated manifests are sent to a third party.